src/java/htsjdk/samtools/reference/AbstractFastaSequenceFile.java

```diff
@@ -69,7 +73,38 @@
         }
     }
 
+    /**
+     * Finds and loads the sequence file dictionary.
+     * @param file Fasta file to read.  Also acts as a prefix for supporting files.
+     */
+    AbstractFastaSequenceFile(final Path file) {
```

Comment:
Also could you change the parameter name to `path` please - it's confusing have `Path` objects named `file`.